site/en/docs/privacy-sandbox/chips/index.md

```diff
@@ -0,0 +1,139 @@
+
+Under this proposal, when a user visits site A and embedded content from site C sets a cookie with the `Partitioned `attribute, the cookie is saved in a partitioned jar designated only for cookies that the site C sets when it's embedded on site A. The browser would only send that cookie when the top-level site is A.   
+
+When the user visits a new site, for example site B, an embedded C frame would not receive the cookie that was set when C was embedded in site A. 
+
+{% Img src="image/vgdbNJBYHma2o62ZqYmcnkq3j0o1/PODInBIXZrgGeUoFhipj.png", alt="ALT_TEXT_HERE", width="800", height="443" %}
+
+If a user visits site C as a top level website, the partitioned cookie that C set when it was embedded in A will not be sent in that request either.
+
+{% Img src="image/vgdbNJBYHma2o62ZqYmcnkq3j0o1/FaQYaZyTsAxCm8GvvVc8.png", alt="ALT_TEXT_HERE", width="800", height="608" %}
+
+### Why it's important to opt into cookie partitioning
+
+As browsers are phasing out unpartitioned third-party cookies, a couple of other approaches to partitioning have been attempted.
+
+Firefox announced that they are [partitioning all third-party cookies by default](https://hacks.mozilla.org/2021/02/introducing-state-partitioning/) in their ETP Strict mode, so all cross-site cookies are partitioned by the top-level site.   
+However, partitioning cookies without a third-party opt-in can lead to unexpected bugs, since some third party services have built servers with the expectation of an unpartitioned third-party cookie.
+
+[Safari previously tried partitioning cookies based on heuristics](https://webkit.org/blog/8613/intelligent-tracking-prevention-2-1/), but eventually chose to block them altogether, citing developer confusion as one of the reasons.
+
+What sets CHIPS apart from existing implementations of partitioned cookies is the third-party opt-in. Cookies must be set with a new attribute in order to be sent on cross-party requests once (unpartitioned) third-party cookies are obsoleted.
+
+While third-party cookies still exist, the `Partitioned` attribute provides an opt-in to a more restrictive, more secure type of cookie behavior. CHIPS is an important step to help services make a smooth transition to a future without third-party cookies.
+
+### CHIPS design details
+
+#### Partitioning model
+
+Today, cookies are keyed on the hostname or domain of the site that set them, that is, their _host key_.   
+
+For example, for cookies from `https://support.chat.example`, the host key is `("support.chat.example")`.  
+
+Under CHIPS, cookies that opt into partitioning will be double-keyed on their host key and _partition key_.   
+
+A _cookie's partition key_ is the site ([scheme and registrable domain](https://web.dev/same-site-same-origin/#%22schemeful-same-site%22)) of the top-level URL the browser was visiting at the start of the request to the endpoint that set the cookie.  
```

Comment:
```suggestion
A _cookie's partition key_ is the site ([scheme and registrable domain](https://web.dev/same-site-same-origin/#%22schemeful-same-site%22)) of the top-level URL the browser is visiting at the start of a request to an endpoint that sets the cookie. This means the cookie partition key is shared by a site and its embedded content.
```

Reply:
You define the request's partitioned key, which I think you should still do again, but I believe this benefits from secondary explanation.